### PR TITLE
(#4433) NCIDS Wide Guide Card

### LINKED ISSUE #4433: Story: Create Wide Guide Card for MLP
### Sprint Timing
- Expected to be completed in Sprint 54

### User Needs

- UN 3.1: As a visitor to an MLP, I need the page to be structured so I can locate key information
- UN 3.4: As a visitor to an MLP, I need relevant descriptions for navigational components so I know where I'm going
- UN 3.5: As a visitor to an MLP, I need contextual images for navigational components so I have a better understanding of where I will go
- UN 3.6: As a visitor to an MLP, I need featured content to be framed in a way that makes it feel important, so I don't ignore it due to ad blindness
- UN 3.9: As a visitor to an MLP, I need an interface to links to be obvious and straightforward, so I am able to find the navigation I am looking for quickly
- UN 3.11: As a visitor to an MLP, I need visuals to not take up too much space on the page, so that I can navigate to the information and links I am looking for
- UN 3.17: As an end user/visitor of an MLP, I need the page and its components to be organized so that I can easily scan the page to quickly find information
- UN 3.18: As an end user/visitor of an MLP, I need the page and its components to be organized so that I can easily interact with elements on the page​
- UN 3.25: As a stakeholder of an MLP,  I need callout boxes and feature cards that allow me to highlight featured information so that I can direct visitors to important information on the site
- UN 3.26: As a stakeholder of an MLP, I need more interactive options that would create an opportunity for greater audience engagement.​

### Requirements

- The content editor shall be able to upload a new 1x1 photo to the component
- The component shall have an optional title to be used when the component is not the first one on the page
- If the title is set, it shall be an h2 in the hierarchy for accessibility
- The title shall have the same font size as all headings on Guide Cards from the Design System. Note: The Guide Cards on the Digital Platform have not been updated to reflect the decision for the correct size. Please see the specifications section for the appropriate values.
- The body text shall have the same font size as all Guide Cards from the Design System. Note: The Guide Cards on the Digital Platform have not been updated to reflect the decision for the correct size. Please see the specifications section for the appropriate values.
- The body text shall be able to be bolded and/or italicized
- The component shall allow the addition of up to three (3) call-to-action buttons
- The character limit for the Wide Guide Card will be handled through guidance with the following information:
  - 1 line Title: ~30 characters. Description: ~354 characters
  - 2 line Title: ~60 characters. Description: ~300 characters. 3 CTAs
  - 3 line Title: ~78 characters. Description: ~247 characters. 3 CTAs

### Specifications
-  The optional title font will be token size xl/24px
- The body font will be token size xs/16px
- The image upload will only accept images with an aspect ratio of 1:1 and a minimum upload size of 1060px x 1060px
- The component will exactly match the comp
- The Drupal interface will exactly match the content model
- The image is not required
  - If the image is not set, a placeholder image will be put in its place.
- The Description field will use the Simple WYSIWYG (bold and italics only)
- The maximum number of call-to-action buttons buttons will be three (3)
- There will be no minimum number of call-to-action buttons buttons, but the guidance will be to have at least two (2)
- If the title, description, and call-to-action buttons take up more vertical space than the image is tall, then the left side of the Wide Guide Card (the box with those components) will vertically expand to contain the text. The image will remain the same size, and will stay top and right aligned.

### Analytics Specifications
On click of a link button within a summary box, an EDDL event shall be raised with the linkName and event of: MLP:WideGuideCard:LinkClick.

This click event shall send the following information as data elements to EDDL:

_Information about the page that was clicked:_

- location: This value shall be set to 'Body'
- pageType: This value shall be set to match the content type of the page. In this case: 'cgvMiniLanding'
- pageTemplate: This value should be set to match the template of the page. In this case: 'ncids_default'

_Location within the page of the component clicked:_

- pageRows: This value shall be set to match the total number of rows on the page overall
- pageRowIndex: This value shall be set to the index of the row from the component clicked within the page
- pageRowCols: This value shall be set to the number of columns within the row of the clicked component. (Will be set to 0 if no column component is present.)
- pageRowColIndex: This value shall be set to the index of the selected column within the selected row from the link click. (Will be set to 0 if no column component is present.)
- pageRowVariant - If a component is not in the two column container, this should pass as Not Defined. If a component is in the two column container, this value should be captured as TwoColumn50:50 or TwoColumn66:33.
- containerItems: This value shall be set to the number of items within a container or row. For Wide Guide Card, this will be 1.
- containerItemIndex: This value shall be set to the position of the clicked item within the container or row. For Wide Guide Card, this will be 1.
- (For examples of how these values might be set see: https://app.mural.co/t/publicissapient7269/m/publicissapient7269/1721221635530/6d43c52f53d2171d5bc1fd66707d5c4496a4a867?sender=u68cf40f39616fa5333f84484)

_Information about the component clicked:_

- componentType: This is the name of the component and should be set to 'Wide Guide Card'
- componentTheme: As this component does not currently have theme options, this value should be set to 'Not Defined'
- componentVariant: As this component does not currently have variation options, this value should be set to 'Not Defined'
- linkText: This value shall be set to the text content of the button link clicked. This value should be truncated after 50 characters.
- title: This value should be the heading of the Wide Guide Card. If no heading is set for the Wide Guide Card, this value will pass 'Not Defined'. This value should be truncated after 50 characters.
- linkType: This should be based on the type of link. The values will be:
  - `External` - This is for external links outside of the host web site. Conditions that match this:
    - If the url starts with `http://` or `https://` AND the hostname does not match the current hostname in the browser. (e.g., the href is `https://www.cancer.gov/foo` and the visitor is on the web page `https://dceg.cancer.gov/bar`)
    - `Internal` - This is for managed and unmanaged links within the same host. Conditions that match this:
      - If the url starts with `http://` or `https://` AND the hostname match the current hostname in the browser. (e.g., the href is `https://www.cancer.gov/foo` and the visitor is on the web page `https://www.cancer.gov/bar`)
- linkArea: This will pass 'Button' based on the link button clicked.
- totalLinks: This will count the total number of links (call-to-action buttons) in the Wide Guide Card
- linkPosition: This will count the index position of the link in the Wide Guide Card as a data attribute.

### Resources

- [Wide Guide Card Comp](https://www.figma.com/design/GCOniv8OTyj9OC1udKwo6S/Mini-Landing-Pages?node-id=4403-29459&m=dev)
- [Drupal Content Model](https://github.com/NCIOCPL/cgov-digital-platform/wiki/Paragraph-Type-NCIDS-Wide-Guide-Card)
- Image for Real Content 

Image

- Image for Test Content WGC 1:

Image

#### Scenarios

- **Real Content**
  - Create an example page with the title, "Clinical Trials Information for Patients and Caregivers", with a url of /test//research/participate/clinical-trials, and make the page exactly match the [comp here](https://www.figma.com/design/GCOniv8OTyj9OC1udKwo6S/Mini-Landing-Pages?node-id=5955-38950&node-type=FRAME&t=hkFdv0Ac4a12zcIi-0).

- **Test Content**
  - Create an example page [based on the comps here](https://www.figma.com/design/GCOniv8OTyj9OC1udKwo6S/Mini-Landing-Pages?node-id=7138-23503&t=bZB0thkS1h32b54N-1) with the title, "Test Page for Wide Guide Card", and use a url of /test/ncids-mini-landing-wide-guide-card, please add the following:
    - Title: "Test Page for Wide Guide Card"
    - Wide Guide Card 1: 
      - No WGC heading. 
      - Image: See image in Resources section. 
      - Description: "This is a description that is five hundred characters long and will be followed with 3 call-to-action buttons. Lorem ipsum odor amet, consectetuer adipiscing elit. **Augue tristique dolor id metus turpis mauris varius. Eu montes diam elementum leo finibus dolor.** _Vivamus tortor lacus ridiculus vel erat ornare nam._ **_Habitant puvinar sodales ipsum metus purus._** Fusce ornare rutrum quisque lacus nam nascetur vel. Mi nostra ut consequat habitant vehicula. Maecenas primis leo platea odio mi mi. Ad suspen" 
      - CTA 1: "[Internal Link to Coping with Cancer]"(https://www.cancer.gov/about-cancer/coping). 
      - CTA 2: "[External Link to Google](https://www.google.com/)". 
      - CTA 3: "[Internal to Cancer Currents Blog](https://www.cancer.gov/news-events/cancer-currents-blog)". 
    - Feature Card Row: 
      - Heading "Optional Feature Card Row Heading". 
      - FC 1: 
        - Title: "Lorem ipsum dolor sit amet consectetur". 
        - Description: "Success stories blurb. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor." 
        - [Link](https://www.cancer.gov/types). 
      - FC 2: 
        - Title: "Lorem ipsum dolor sit amet consectetur adipiscing elit". 
        - Description: "Blurb for feature card two. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor." 
        - [Link](https://www.cancer.gov/grants-training). 
      - FC 3: 
        - Title: "Lorem ipsum dolor sit". 
        - Description: "Feature card three blurb. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor." 
        - [Link](https://www.cancer.gov/news-events).
    - Wide Guide Card 2: 
      - Image: No image (should default to the grey mountains 'no image' picture)
      - WGC Heading: "Optional Heading for Wide Guide Card". 
      - Description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam lobortis eros ut eros rutrum, et interdumwow dolor tincidunt. Aliquam cursus risus in mattis maximus. Pellentesque tristique mi non velit pretium pretium. Morbi varius vestibulum elit volutpat. Sed pulvinar placerat. Wllgwyng yllgogerych". 
      - CTA 1: "Cancer Homepage" link to homepage [here](https://www.cancer.gov/). 
    - Wide Guide Card 3:
      - Image: No Image (should default to the grey mountains 'no image' picture)
      - WGC Heading: "This is a Very Long Heading That Will Cause the Wide Guide Card to Look Badly, but This Scenario Will be Prevented by Guidance"
      - Description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam lobortis eros ut eros rutrum, et interdumwow dolor tincidunt. Aliquam cursus risus in mattis maximus. Pellentesque tristique mi non velit pretium pretium. Morbi varius vestibulum elit volutpat. Sed pulvinar placerat. Wllgwyng yllgogerych. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam lobortis eros ut eros rutrum, et interdumwow dolor tincidunt. Aliquam cursus risus in mattis maximus. Pellentesque tristique mi non velit pretium pretium. Morbi varius vestibulum elit volutpat. Sed pulvinar placerat. Wllgwyng yllgogerych. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam lobortis eros ut eros rutrum, et interdumwow dolor tincidunt. Aliquam cursus risus in mattis maximus. Pellentesque tristique mi non velit pretium pretium. Morbi varius vestibulum elit volutpat. Sed pulvinar placerat. Wllgwyng yllgogerych.".
      - CTA 1: "[This is an internal link with a very long title to test for analytics, and the character count will cut off at 50 characters](https://www.cancer.gov/types)"
      - CTA 2: "[This is an External Link with a very long title to test for analytics, and the character count will cut off at 50 characters](https://www.merriam-webster.com/dictionary/analytics)"

### DoR Tasks

- [x] Content Model Drafted
- [x] Scenarios created for YAML
- [x] Acceptance Criteria Drafted
  - [ ] Analytics Acceptance Criteria Drafted
- [ ] Content Model Reviewed
  - [ ] Developer Review
  - [x] IA Review
- [ ] Content Model Approved
  - [ ]  Developer Approval
  - [x] IA Approval
- [ ] Scenarios Reviewed
  - [ ] Developer Review
  - [x] IA Review
  - [ ] Analytics Review
- [ ] Scenarios Approved
  - [ ] Developer Approval
  - [x] IA Approval
  - [ ] Analytics Approval
- [ ] Acceptance Criteria Reviewed
  - [ ] Developer Review
  - [x] IA Review
  - [ ] Analytics Review
- [ ] Acceptance Criteria Approved
  - [ ] Developer Approval
  - [x] IA Approval
  - [ ] Analytics Approval

### DoD Tasks

- [ ] Governance Drafted
- [ ] PR Review (see PR for review and comments)
- [ ] Client Review with Demo

### Acceptance Criteria
**CMS User Experience**
```gherkin
Scenario: NCIDS Wide Guide Card paragraph can be added to Mini Landing Page content type
  Given the user is logged into the CMS as an editor
    And the user is editing a Mini Landing Page content item
    And the Mini Landing Page is using the NCIDS style
  When the user expands the dropdown to add a paragraph to Contents
  Then the dropdown displays "Add NCIDS Wide Guide Card"
  When the user selects "Add NCIDS Wide Guide Card"
  Then the fields for an "NCIDS Wide Guide Card" paragraph appear
    And the first field is the "Title" field
    And the field is optional
    And it is a text field
    And the next field is the "Image" field
    And the field is optional
    And it is a media upload field
    And the next field is the "Description" field
    And the field is required
    And it is a Simple WYSIWYG field (text editor with bold and italics)
    And the next field is the "Links for Wide Guide Card Buttons" field
    And the field is required
    And it is a paragraph with options: "Add NCIDS Link Button Internal" and "Add NCIDS Link Button External"
    And the dropdown button defaults to "Add NCIDS Link Button Internal"
    And the user is able to add up to three (3) links
  When the user clicks "Add NCIDS Link Button Internal"
  Then the fields for the NCIDS Link Button Internal appear
  When the user clicks "Add NCIDS Link Button External"
  Then the fields for the NCIDS Link Button External appear
  When a user uploads an image that does not have a 1:1 aspect ratio
  Then the image upload will fail
    And an error will appear stating "Image upload failed. Please upload an image that is at least 1060px x 1060px with a 1:1 aspect ratio"
  When a user uploads an image that is smaller than 1060px x 1060px
  Then the image upload will fail
    And an error will appear stating "Image upload failed. Please upload an image that is at least 1060px x 1060px with a 1:1 aspect ratio".
```

**Front-End User Experience**
```gherkin
Scenario: A site visitor goes to a Mini Landing Page that uses the NCIDS style and has an NCIDS Wide Guide Card on it
  Given the Mini Landing Page uses the NCIDS style
    And the Mini Landing Page has an NCIDS Wide Guide Card on it
    And the NCIDS Wide Guide Card has the text and link buttons from the Scenarios section above
  When the visitor views the Mini Landing Page with the NCIDS style and the NCIDS Wide Guide Card
  Then the visitor will see the NCIDS Wide Guide Card on the page
    And it will exactly match the comps
    And for the scenario with a Wide Guide Card with a heading, the heading will be an h2 in the page hierarchy, with a token size of xl/24px
  When the user clicks on a link button in the NCIDS Wide Guide Card
  Then they will be redirected to the linked page
  When the content editor did not add a heading to the NCIDS Wide Guide Card
  Then no heading will appear in the NCIDS Wide Guide Card
  When the content editor did not add an image to the NCIDS Wide Guide Card
  Then a placeholder image will appear on the NCIDS Wide Guide Card
```

**Analytics ACs** (Work In Progress)
Wide Guide Card MLP – No Title, Internal link click event - managed
```gherkin
Scenario: Wide Guide Card MLP – No Title, Internal link click event - managed
  Given user is navigating to "/test/ncids-mini-landing-wide-guide-card"
    And browser waits
When user clicks in the 1st Wide Guide Card on the “Coping with Cancer” button in the 1st position.
        Then page click request is sent
        And browser waits
        And the following parameters should be captured
    And the Mini Landing Page has an NCIDS Wide Guide Card
  When the user clicks on a link in the NCIDS Wide Guide Card
            | parameter | value                                                          |
            | prop4     | D=pev1                                                         |
            | prop8     | english                                                        |
            | prop57    | D=v64                                                          |
            | prop58    | D=v65                                                          |
            | prop59    | D=v66                                                          |
            | prop60    | D=c67                                                          |
            | prop67    | D=pageName                                                     |
            | prop68    | D=v68                                                          |
            | evar2     | D=c8                                                           |
            | evar61    | cgvMiniLanding\|ncids_default                                  |
            | evar64    | Internal\|Not Defined\|Coping with Cancer\                                 |
            | evar65    | Wide Guide Card\|Not Defined\| Not Defined                      |
            | evar66    | 4\|1\|0\|0\|1\|1\|Not Defined                                            |
            | evar67    | Button\|3\|1                                                    |
            | evar68    | Body                                                           |
            | pageName  | {CANONICAL_HOST}/test/ncids-mini-landing-summary-box |
            | pageURL   | {PROTOCOL}://{CANONICAL_HOST}/test/ncids-mini-landing-summary-box |
            | event112  |                                                                |
            | pev2      | MLP:WideGuideCard:LinkClick                                      |
            | linkType  | lnk_o                                                          | 

Wide Guide Card MLP - Title, Managed Internal link click
```gherkin
Scenario: Wide Guide Card MLP – No Title, Internal link click event - managed
  Given user is navigating to "/test/ncids-mini-landing-wide-guide-card"
    And browser waits
When user clicks on Wide Guide Card in the third row on the page on the “Cancer Homepage” button in the 1st position.
        Then page click request is sent
        And browser waits
        And the following parameters should be captured
    And the Mini Landing Page has an NCIDS Wide Guide Card
  When the user clicks on a link in the NCIDS Wide Guide Card
            | parameter | value                                                          |
            | prop4     | D=pev1                                                         |
            | prop8     | english                                                        |
            | prop57    | D=v64                                                          |
            | prop58    | D=v65                                                          |
            | prop59    | D=v66                                                          |
            | prop60    | D=c67                                                          |
            | prop67    | D=pageName                                                     |
            | prop68    | D=v68                                                          |
            | evar2     | D=c8                                                           |
            | evar61    | cgvMiniLanding\|ncids_default                                  |
            | evar64    | Internal\|Optional Heading for Wide Guide Card\|Cancer Homepage\                                 |
            | evar65    | Wide Guide Card\|Not Defined\| Not Defined                      |
            | evar66    | 4\|3\|0\|0\|1\|1|Not Defined                                               |
            | evar67    | Button\|1\|1                                                    |
            | evar68    | Body                                                           |
            | pageName  | {CANONICAL_HOST}/test/ncids-mini-landing-summary-box |
            | pageURL   | {PROTOCOL}://{CANONICAL_HOST}/test/ncids-mini-landing-summary-box |
            | event112  |                                                                |
            | pev2      | MLP:WideGuideCard:LinkClick                                      |
            | linkType  | lnk_o                                                          | 

Wide Guide Card MLP - Long title, Long button text, external link click
```gherkin
Scenario: Wide Guide Card MLP – Long title, Long button text, external link click
  Given user is navigating to "/test/ncids-mini-landing-wide-guide-card"
    And browser waits
When user clicks in the Wide Guide Card in the 4th row on the page on the second CTA button with the long button text.
        Then page click request is sent
        And browser waits
        And the following parameters should be captured
    And the Mini Landing Page has an NCIDS Wide Guide Card
  When the user clicks on a link in the NCIDS Wide Guide Card
            | parameter | value                                                          |
            | prop4     | D=pev1                                                         |
            | prop8     | english                                                        |
            | prop57    | D=v64                                                          |
            | prop58    | D=v65                                                          |
            | prop59    | D=v66                                                          |
            | prop60    | D=c67                                                          |
            | prop67    | D=pageName                                                     |
            | prop68    | D=v68                                                          |
            | evar2     | D=c8                                                           |
            | evar61    | cgvMiniLanding\|ncids_default                                  |
            | evar64    | Externall\|This is a Very Long Heading That Will Cause the Wide Guide Car\|This is an External Link with a very long title to test for ana\                                 |
            | evar65    | Wide Guide Card\|Not Defined\| Not Defined                      |
            | evar66    | 4\|4\|0\|0\|1\|1|Not Defined                                               |
            | evar67    | Button\|2\|2                                                    |
            | evar68    | Body                                                           |
            | pageName  | {CANONICAL_HOST}/test/ncids-mini-landing-summary-box |
            | pageURL   | {PROTOCOL}://{CANONICAL_HOST}/test/ncids-mini-landing-summary-box |
            | event112  |                                                                |
            | pev2      | MLP:WideGuideCard:LinkClick                                      |
            | linkType  | lnk_o                                                          |

### PR DESCRIPTION
**Pull request by dev-rana-publicis**
Closes #4433
ODE: https://ncigovcdode835.prod.acquia-sites.com/

### CODE REVIEW

**Comment by blilianyu** [comment]
@dev-rana-publicis Looks great but a couple of things!
1. The size of the image in Figma is 530x530 and on the ODE it's 497x497. Can we increase the size or is that going to break something?
2. Can we add a min-height to the body box? 450 if image is 497 and 482 if we can change the image size to 530x530?

**Comment by dlescarbeau** [comment]
Adjustments made per feedback:
- Content box height now sets the height of the container, image now scales with height of container rather than having a set minimum width
- Border radius updated from 8px to 4px
- z-index updated so wide guide card is underneath Back to Top on mobile

**Comment by lburack** [comment]
When an image is uploaded, it displays at an incredibly large size in the Drupal form. Are we able to make the image display smaller?

File: docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/lib/components/cgdp-wide-guide-card/__tests__/cgdp-wide-guide-card.dom.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
+export const cgdpWideGuideCardDom = `
+<div class="usa-section" data-eddl-landing-row="">
```
Comment (bryanpizzillo) [inline docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/lib/components/cgdp-wide-guide-card/__tests__/cgdp-wide-guide-card.dom.ts:2]: Check this HTML for outdated cgdp-wide-guide-card references. Also it should match the template HTML. The image is being set as a background image and this content has a cgdp-wide-guide-card__image element.

File: docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/lib/components/cgdp-wide-guide-card/__tests__/cgdp-wide-guide-card.bad.dom.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
+export const cgdpWideGuideCardBadDom = `
+<div class="usa-section" data-eddl-landing-row="">
```
Comment (bryanpizzillo) [inline docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/lib/components/cgdp-wide-guide-card/__tests__/cgdp-wide-guide-card.bad.dom.ts:2]: Please cleanup the HTML here.

**Review by bryanpizzillo — CHANGES_REQUESTED** [review]
Almost...

File: docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/mini_landing/paragraph--ncids-wide-guide-card.html.twig [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+	<div class="nci-guide-card nci-guide-card--cgdp-wide" style="background-image: url({{ image_url }})">
+		<div class="nci-guide-card__wrapper">
+			<div class="nci-guide-card__body">
+				{% if content.field_container_heading[0] %}
+					<h2 class="nci-guide-card__title">{{ content.field_container_heading }}</h2>
+				{% endif %}
+				<div class="nci-guide-card__description usa-prose">
+					{{ content.field_html_content }}
+				</div>
+				<ul class="nci-card__button-group">
```
Comment (bryanpizzillo) [inline docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/mini_landing/paragraph--ncids-wide-guide-card.html.twig:20]: Gah... this should be nci-guide-card__button-group

Comment (dlescarbeau) [reply docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/mini_landing/paragraph--ncids-wide-guide-card.html.twig:20]: This actually is correct, there is no nci-guide-card__button-group

**Comment by lburack** [comment]
> Drupal form feedback:
> 
> * When an image is uploaded, it displays at an incredibly large size in the Drupal form. Are we able to make the image display smaller?
> * CTA buttons should be collapsed when there is more than one added (captured in [Paragraph interface for Claro was not updated for Mini Landing Page template #4401](https://github.com/NCIOCPL/cgov-digital-platform/issues/4401))

- The image size now looks good. Thanks!
- It looks like the CTA buttons can be manually collapsed by the user (there is a collapse option). I'm not sure if #4401 is correct that this Claro feature wasn't updated for the MLP template or the automatic collapse just wasn't added here, b/c it's manually available.

**Comment by bennettcc** [comment]
Approved by the Fed team (Anna, Alex, Jorge) on 10/11

**Review by andyvanavery31 — APPROVED** [review]
Passes Product review and has been approved by the clients.

**Review by bryanpizzillo — APPROVED** [review]

**Pull request merged**